tests/openvino/test_diffusion.py

```diff
@@ -820,7 +840,7 @@ def test_image_reproducibility(self, model_arch: str):
         pipeline = self.OVMODEL_CLASS.from_pretrained(MODEL_NAMES[model_arch])
 
         height, width, batch_size = 64, 64, 1
-        inputs = self.generate_inputs(height=height, width=width, batch_size=batch_size)
+        inputs = self.generate_inputs(height=height, width=width, batch_size=batch_size, model_arch=model_arch)
 
```

Comment:
also @eaidova would you mind adding a test for the CLI as well ? https://github.com/huggingface/optimum-intel/blob/main/tests/openvino/test_exporters_cli.py#L67C5-L67C28